public/assets/jquery-ui/effect-bounce.js

```diff
@@ -8,7 +8,7 @@
  *
  * http://api.jqueryui.com/category/effects-core/
  */
-!function(e){"function"==typeof define&&define.amd?define(["jquery"],e):e(jQuery)}(function(e){var t="ui-effects-",n=e;/*!
+!function(t){"function"==typeof define&&define.amd?define(["jquery"],t):t(jQuery)}(function(t){var e="ui-effects-",i=t;/*!
```

Comment:
__[jshint]__
* Expected an assignment or function call and instead saw an expression.

<!-- Comment by [SideCI](https://sideci.com) -->